.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: php
+php:
+  - 5.3
+  - 5.4
+  - 5.5
+  - 5.6
+#  - 7.0
+#  - hhvm
+
+env: COVERAGE=0
+
+matrix:
+  include:
+    - php: 5.6
+      env: COVERAGE=1
+
+before_script:
+  - if [[ "$COVERAGE" -eq 0 ]]; then composer install; else composer require satooshi/php-coveralls --dev; fi
+
+script:
+  - if [[ "$COVERAGE" -eq 0 ]]; then vendor/bin/phpunit; else mkdir -p build/logs; vendor/bin/phpunit --coverage-clover=build/logs/clover.xml; fi
+
+after_script:
```

Comment:
In addition, in my personal fork, I setup Travis. PHP 5.3 is a no-go, as we use bracket notation for arrays in the client library. However, PHP7 and HHVM were successful! 

With that said, please remove PHP5.3, and uncomment 7.0 and HHVM.